content/docs/i18n.mdx

```diff
@@ -24,9 +24,10 @@ const createTranslator = (locale) => (key, defaultMessage) => {
 
 const [locale, setLocale] = useState<'de'|'en'>('de');
 const translation = useMemo(() => createTranslator(locale), [locale]);
+const i18n = {locale: locale, translate: translation};
 
 <JsonForms
-  i18n={{locale: locale, translate: translation}}
+  i18n={i18n}
```

Comment:
This change is meaningless as we still construct a new object with each render call. However for the `i18n` property this is fine. So you can either revert this or keep it, both are fine.